docs/METHODS_RPC.md

```diff
@@ -0,0 +1,107 @@
+## RPC
+
+_The following RPC methods are not part of the default Substrate runtime. They are hard-coded in Substrate._
+
+---
+
+### author
+
+_Authoring of network items_
+
+▸ **pendingExtrinsics**(): `PendingExtrinsics`
+- **summary**: Returns all pending extrinsics, potentially grouped by sender
+
+▸ **submitExtrinsic**(extrinsic: `Extrinsic`): `Hash`
+- **summary**: Submit a fully formatted extrinsic for block inclusion
+- **isSigned**: `true`
+
+---
+
+### chain
+
+_Retrieval of chain data_
+
+▸ **getBlock**(hash: `Hash`): `SignedBlock`
+- **summary**: Get header and body of a relay chain block
+
+▸ **getBlockHash**(blockNumber?: `BlockNumber`): `Hash`
+- **summary**: Get the block hash for a specific block
+
+▸ **getHead**(): `Hash`
+- **summary**: Retrieves the best headerHash
+
+▸ **getHeader**(hash: `Hash`): `Header`
+- **summary**: Retrieves the header for a specific block
+
+▸ **getRuntimeVersion**(): `RuntimeVersion`
+- **summary**: Get the runtime version
+
+▸ **getRuntimeVersionAt**(hash: `Hash`): `RuntimeVersion`
+- **summary**: Get the runtime version at a specific block
+
+▸ **newHead().subscribe**(`<CALLBACK>`): `Header`
```

Comment:
Generate all these as well via script, it is outdated from what is in master atm.